targets/targets.json

```diff
@@ -1280,7 +1280,8 @@
         "device_has_add": [
             "CAN",
             "SERIAL_ASYNCH",
-            "FLASH"
+            "FLASH",
+            "USBDEVICE"
```

Comment:
No, we can't add it there, as USBDEVICEmeans you have an USB connector.
This can be only defined in each target (in you custom_target.json)

Reply:
reverted